index.js

```diff
@@ -1,18 +1,51 @@
 /***** Deliverable 1 *****/
+
 const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+console.log("Here's your header: ", header)
 
 
 /***** Deliverable 2 *****/
 
+header.style.color = 'pink'
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+
+console.log('This is what the traveler object looks like: ', traveler)
+let travelerName = document.querySelector("h2")
+let travelerNickName = document.querySelector("em")
+let travelerImg = document.querySelector("img") 
+
+travelerName.textContent = traveler.name
+travelerNickName.textContent = traveler.nickname
+travelerImg.src = traveler.photo
+travelerImg.alt = traveler.name
 
 
 /***** Deliverable 4 *****/
 
+traveler.animalSightings.forEach( function(obj) {
```

Comment:
It would be better to use a more descriptive variable name than `obj` here - like `animalObj` or `sightingObj`.